index.html

```diff
@@ -0,0 +1,147 @@
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    <title>Homework HTML/CSS</title>
+    <style>
+        body {
+            background-color: #ffeead;
+        }
+
+        .container {
+            display: grid;
+            grid-gap: 3px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: 50px auto 50px;
+            height: 100%;
+        }
+
+        .header {
+            grid-column: 2 / -1;
+
+            background-color: #ff706a;
+
+            display: flex;
+        }
+
+        .m-1 {
+            margin: 15px;
+        }
+
+        .header > div:first-child {
+            display: flex;
+        }
+
+        .header > div:last-child {
+            margin-left: auto;
+        }
+
+        .logo {
+            grid-row: 1 / 2;
+
+            background-color: #96ceb4;
+        }
+
+        .menu {
+            grid-row: 2 / 3;
+
+            background-color: #00cc9a;
+
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: space-between;
+        }
+
+        div {
+            font-size: 1.5vw;
+            font-weight: bold;
+        }
+
+        .menu div {
+            font-size: 1vw;
+            margin: 10px;
+        }
+
+        .content {
+            grid-column: 2 / -1;
+
+            background-color: #fecc5c;
+
+            display: grid;
+            padding: 20px;
+            grid-gap: 5px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: auto auto;
+        }
+
+        .content1 {
+            background-color: #d6ccd3;
+
+            grid-column: 1 / 4;
+        }
+
+        .content2 {
+            background-color: #eca1a7;
+
+            grid-column: 4 / 9;
+        }
+
+        .content3 {
+            background-color: #bdcebf;
+
+            grid-column: 1 / 9;
+        }
+
+        .content4 {
+            background-color: #ada398;
+
+            grid-column: 9 / -1;
+            grid-row: 1 / 3;
+        }
+
+        .footer {
+            grid-column: 1 / -1;
+
+            background-color: #df9edf;
+        }
+
+        .text-centered {
+            color: #ffeead;
+
+            display:flex;
+            align-items: center; /* Vertical center alignment */
+            justify-content: center; /* Horizontal center alignment */
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
```

Comment:
There is no sense to wrap left part of header content into a separate div. You could do like this:
` <div class="header">
            <div class="text-centered m-1">HOME</div>
            <div class="text-centered m-1">ABOUT</div>
            <div class="text-centered m-1">GALLERY</div>
            <div class="text-centered m-1">LOG IN</div>
        </div>`
and just add to last header child property "margin-left: auto". It will be positioned on right automatically